tests/cypress/integration/admin.test.js

```diff
@@ -0,0 +1,13 @@
+describe('Admin can login and make sure plugin is activated', () => {
+	it('Can activate plugin if it is deactivated', () => {
+		cy.login();
+		cy.visit('http://localhost:8889/wp-admin/plugins.php');
```

Comment:
You can put `http://localhost:8889` into `cypress.json` under `baseUrl` and then you can just supply a URL starting with the path (or just use your `visitAdminPage()` Cypress command).